docs/source/development.rst

```diff
@@ -0,0 +1,19 @@
+.. _development:
+
+========================
+Contributing to Borealis
+========================
+
+Development on the Borealis software is both appreciated and encouraged. If you have new features you would like added,
+please reach out via email to superdarn@usask.ca and we would be happy to work with you to implement your request.
+
+----------------------
+Contribution Standards
+----------------------
+
+``pre-commit`` hooks are used to enforce coding standards for this project. To use ``pre-commit`` in your environment,
+install ``cppcheck`` and ``clang-format`` with your native package manager and install ``borealis[dev]`` in your
```

Comment:
What does `borealis[dev]` mean?

Reply:
It installs the optional dependencies specified with the `dev` keyword in the pyproject.toml file